src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1400,6 +1400,7 @@ private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
 
         protected virtual void EnterParameter(ParameterSymbol parameter)
         {
+            Visit(parameter.ExplicitDefaultExpression);
```

Comment:
>Visit(parameter.ExplicitDefaultExpression); [](start = 12, length = 43)

Could this have effect on Region Analysis API? For example, could locals declared in the default value make their way to the declared set and other sets collected for the method body?